app/components/form/instance-table.js

```diff
@@ -41,16 +41,45 @@ export default class InstanceTableComponent extends Component {
     }
   }
 
+  @action
   async onInit() {
```

Comment:
This seems more complicated than it should be, I think you could rely more on ember to track the whole pagination etc. but I don't really have time to investigate it now.

Reply:
or maybe not since we don't use resources, I'll have a look at that on Friday.

Reply:
sadly no, the data table assumes the same meta extensions exist like they were done for ember data relations/queries